java/graalvm/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-21.2.0
+FROM ghcr.io/graalvm/graalvm-ce:java11-21.3.0
```

Comment:
Initially, I tried "ol7-java11-21.3.0" but that tag doesn't exist. Now using "java11-21.3.0".

Reply:
This should work: `ghcr.io/graalvm/graalvm-ce:ol7-21.3`

Source: https://github.com/graalvm/container/pkgs/container/graalvm-ce

Reply:
> ghcr.io/graalvm/graalvm-ce:ol7-21.3

Note, this is Java 17.

Reply:
Updated to "ol7-java11-21.3". dzou@ told me that this "ol" stands for Oracle Linux and is required to setup gcloud tool.

Reply:
Well, the funny thing is, `:java11-21.3.0` is still Oracle Linux. A newer version actually.